packages/itwin/measure-tools/package.json

```diff
@@ -43,13 +44,13 @@
   },
   "peerDependencies": {
     "@itwin/appui-abstract": "^4.7.0",
-    "@itwin/appui-react": "^4.17.0",
-    "@itwin/components-react": "^4.17.0",
+    "@itwin/appui-react": "^4.17.0 || ^5.0.0",
```

Comment:
thoughts on dropping appui 4.17?

Reply:
Pineapple still uses it for now, and it seems quick to drop support for 4.17.x given the short amount of time everyone had to update to 5.x. I'd vote to keep like this for a few weeks at least.